packages/odyssey-react/.npmignore

```diff
@@ -0,0 +1,2 @@
+*
+!dist/**
```

Comment:
Is the README included in this output? If not, let's consider adding it for devs to reference when combing through their `node_modules` 😄

Reply:
README is never ignored so it will be included (but we dont' have one for this package specifically)